src/routes/admin/_admin.transactions.tsx

```diff
@@ -82,7 +84,7 @@ export function Transactions() {
             <TabsTrigger value="all">All</TabsTrigger>
           </TabsList>
           <div className="flex">
-            <RecordTxDialogAdmin />
+            {showRecordTransaction && <RecordTxDialogAdmin />}
```

Comment:
if you need to hide `Record transaction` button, you can remove this component completely from `_admin.transactions.tsx`

Reply:
I don't agree, 
if we will remove this component, after some time, no one would remember that it is already there, and then we can actually remove the entire feature from code.

Reply:
Ok.
PS: I like the idea with `display: none` or `visibility: hidden` more :blush:

Reply:
And how will you show that this change is on purpose? 

For me `display:none` has the same "value" as commented code. There is no value in terms of explaining why we hide specific feature.